podman/api/tar_utils.py

```diff
@@ -1,22 +1,22 @@
 """Utility functions for working with tarball's."""
-import os
 import pathlib
 import random
 import shutil
-import sys
 import tarfile
 import tempfile
 from fnmatch import fnmatch
 from typing import BinaryIO, List, Optional
 
+import sys
+
 
 def prepare_dockerignore(anchor: str) -> List[str]:
     """Read .dockerignore and return exclude list."""
-    file = os.path.join(anchor, ".dockerignore")
-    if not os.path.exists(file):
+    ignore = pathlib.Path(anchor) / ".dockerignore"
```

Comment:
oh cool! didn't know you could do that!

Reply:
The `pathlib` has a lot of code sugar in it. :+1: